### ISSUE TITLE
Предложение: Изменить общий шаблон приложения

### ISSUE THREAD

**Issue by Oleg42-prog** [Ivan-Nebogatikov/ChineseDictionary#43]
В этой задаче предлагается изменить дизайн тех элементов страницы, которые не меняются от страницы к странице.
Сейчас у нас используется стандартный шаблон и это плохо, так как у людей, которые уже сталкивались с приложениями на Blazor или Bootstrap, может сразу сложиться плохое впечатление, потому что все новички используют стандартный шаблон => много плохих приложений с таким шаблоном => стандартный внешний вид сразу настраивает пользователя негативно вследствие отрицательного опыта.

Я предлагаю оставить структуру (т.е. панель меню слева растянутая на всю высоту), но изменить цветовую палитру и шрифт.

Еще мы можем добавить иконку нашего приложения рядом с названием (предварительно сделав ее).

Так же на широкоформатных мониторах остается много пустого места на страничках, поэтому мы можем добавить картинку background (но она должна быть нейтральной и максимально незаметной, чтобы не привлекать к себе внимания, например в серых или полу-прозрачных тонах).

Так как у нас словарь китайского языка, то можем обыграть эту тему в дизайне.
Пример картинок для заднего фона (если будем брать именно эти, то их нужно будет немного отредактировать и порезать):
![—Pngtree—chinese style landscape ink antiquity_938687](https://user-images.githubusercontent.com/67792500/90184416-bf85dd00-ddbd-11ea-8f2d-aab6901d015b.jpg)
![kisspng-china-poster-graphic-design-temple-5a9714e4269714 9814440615198507241581](https://user-images.githubusercontent.com/67792500/90184418-c01e7380-ddbd-11ea-8882-a99aeed12721.png)
![—Pngtree—chinese style ink landscape background_952530](https://user-images.githubusercontent.com/67792500/90184419-c0b70a00-ddbd-11ea-9c21-6ffce2c21aef.jpg)

**Comment by Ivan-Nebogatikov** [comment]
Да, одобряю, и предлагаю этим заняться. Из предложенных картинок мне нравится первая. Либо правая часть последней, так как у нее более нейтральные тона.

**Comment by Ivan-Nebogatikov** [comment]
Еще одна вещь на которую надо обратить внимание -- лицензия на используемые картинки и шрифты. Мы не готовы платить и ограничены ресурсами, предоставляемыми с лицензией, позволяющей использовать их в продуктах.
